packages/wrangler/src/deploy/deploy.ts

```diff
@@ -678,6 +690,10 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			placement,
 			tail_consumers: config.tail_consumers,
 			limits: config.limits,
+			experimental_assets: {
```

Comment:
Should this whole property be undefined if there are no assets?
This could be conditional upon whether `experimentalAssetsJwt` is defined or perhaps on whether `props.experimentalAssets` is defined?

Reply:
Moreover, to keep this new stuff as segregated from the normal modes, we should try to avoid adding any new properties to this object if the new flag is not enabled.

Reply:
Yes it should be, was a bit of a temp solution to let me only touch the `deploy` code path in this PR because it was getting a bit large. Will be doing a fast followup today for `versions upload` and other bits that should clean this up.